aderyn_core/src/detect/experimental/parent_chain.rs

```diff
@@ -79,7 +84,7 @@ mod parent_chain_demo_tests {
             "../tests/contract-playground/out/ParentChainContract.sol/ParentChainContract.json",
         );
 
-        let mut detector = ImmediateParentDemonstrator::default();
+        let mut detector = ParentChainDemonstrator::default();
```

Comment:
We mistakenly used some other detector demonstrator in this file. So I changed it now :)